src/components/ConversationIcon.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import Avatar from 'react-avatar';
+import PropTypes from 'prop-types';
+class ConversationIcon extends PureComponent {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+      conversation,
+      className,
+      extraContent,
```

Comment:
This is not needed as `this.props.children` as the children components. You can use this.

Reply:
Done!